### ISSUE TITLE
Possible Feature Request

### ISSUE THREAD

**Issue by mearmortal** [EDDiscovery/EDDiscovery#2992]
Not sure if this is available now or not, I don't think so, but here it is anyway:
I'd like to be able to view my completed missions, commodity transactions, Combat bonds, whether that be Faction Kill Bonds or Redeemed Vouchers or any related to other forms of killing, like assassinations. Basically all actions completed by me as a summary, of what it was ], who for and what was the gain.  I'd like to be able to do this for my commander, listed either under the faction I'm doing it for, or under the system I'm doing it in for whatever faction.

I'd like to be able to see this for any numbers of days or months, but days mainly.

Do you think this is possible already or something you'll need to look at?

**Comment by robbyxp1** [comment]
You can try the combat panel and set up a campaign, with a faction, and it will tell you all the combat related actions that occurred.

Have a look at that.

rob

**Comment by mearmortal** [comment]
> You can try the combat panel and set up a campaign, with a faction, and it will tell you all the combat related actions that occurred.
> 
> Have a look at that.
> 
> rob

Hi,
    Thanks that lists all the combat bonds, but not nothing of Missions, Commodities. The Factions panel only shows the quantities of all factions and nothing specific, so would be nice if you could have everything a commander has done for a faction or system all in one screen. The missions panel lists all missions, but all this info in different panels is not easy to read as a summary. Do you think this could be possible?

**Comment by robbyxp1** [comment]
So, don't hold you breath on this.  I have the 3dmap to finish, plus any Odyssey stuff to do, and this is a big ask. I'm moving it to project ideas.

Thanks for the input.

**Comment by robbyxp1** [comment]
The faction panel has right clicks, which give more detail on each faction interaction, btw.

**Comment by mearmortal** [comment]
I like the combat option, I can see the kills per faction work I've completed.
I've tried to do the same for Missions, but obviously there is no filter for faction, and I've noticed the date range feature doesn't work, I've specified the range, but still end up with stuff from 4 years ago. I'll attach a screen shot.
I'm using the splitter window, but limited to 2 tabs at a time I believe.
![Screenshot 2021-02-05 134452](https://user-images.githubusercontent.com/16454320/107041426-67ea8c00-67b8-11eb-8af6-36dadcdbcc95.png)
